Dockerfile.dapper

```diff
@@ -1,7 +1,7 @@
 ARG BASE_BRANCH
 FROM quay.io/submariner/shipyard-dapper-base:${BASE_BRANCH}
 
-ENV DAPPER_ENV="REPO MAKEFLAGS TAG QUAY_USERNAME QUAY_PASSWORD GITHUB_REF CLUSTERS_ARGS DEPLOY_ARGS CLEANUP_ARGS IMAGES_ARGS RELEASE_ARGS FOCUS SKIP PLUGIN" \
+ENV DAPPER_ENV="REPO MAKEFLAGS TAG QUAY_USERNAME QUAY_PASSWORD GITHUB_REF CLUSTERS_ARGS DEPLOY_ARGS CLEANUP_ARGS IMAGES_ARGS RELEASE_ARGS FOCUS SKIP PLUGIN BASE_BRANCH" \
```

Comment:
You sure this is needed?

`BASE_BRANCH` is defined outside the dapper scope and inside it, so I don't think it's necessary to pass it in the env.